index.js

```diff
@@ -261,6 +264,40 @@ Rapptor.prototype.setup = function(callback) {
   ], callback);
 };
 
+Rapptor.prototype.callMethod = function(server, method, argv, callback) {
+
+  callback = callback || _.noop;
+
+  var methodFunc = _.get(server.methods, method);
+
+  if(methodFunc) {
+    var funcArgs = argv._;
+
+    for(i=0; i<funcArgs.length; i++){
+      funcArgs[i] = JSON.parse(funcArgs[i]);
+    }
+    
+    funcArgs.push(function(err, data) {
+      if(err) {
+        console.log('An ERROR Occured'.red);
```

Comment:
having `colors` change the string prototype worries me a bit.  I don't think it's worth it